packages/xstate-svelte/src/fsm.ts

```diff
@@ -0,0 +1,53 @@
+import { readable } from 'svelte/store';
+import {
+  createMachine,
+  interpret,
+  EventObject,
+  StateMachine,
+  Typestate
+} from '@xstate/fsm';
+
+interface UseMachineOptions<
+  TContext extends object,
+  TEvent extends EventObject
+> {
+  /**
+   * If provided, will be merged with machine's `actions`.
+   */
+  actions: StateMachine.ActionMap<TContext, TEvent>;
+}
+
+export function useMachine<
+  TContext extends object,
+  TEvent extends EventObject,
+  TTypestate extends Typestate<TContext>
+>(
+  machine: StateMachine.Machine<TContext, TEvent, TTypestate>,
+  options: Partial<UseMachineOptions<TContext, TEvent>> = {}
+) {
+  const resolvedMachine = createMachine(machine.config, {
+    actions: {
+      ...(machine as any)._options.actions,
+      ...options.actions
```

Comment:
Both Vue and React packages are **not** merging those:
https://github.com/davidkpiano/xstate/blob/9fe5cc5c7719e5fa245e883dc4ab60f102c2164f/packages/xstate-vue/src/fsm.ts#L42
https://github.com/davidkpiano/xstate/blob/9fe5cc5c7719e5fa245e883dc4ab60f102c2164f/packages/xstate-react/src/fsm.ts#L57

Reply:
Yes, I saw that (mentioned in first comment). Wouldn't merging be the expected behaviour though?

Reply:
IMHO there is more to consider here - and such a change might be introduced in the future but it should be introduced cohesively, for all integration packages. I would prefer to use the existing logic for this new package for now.

Reply:
Sure. I will make that change and then submit a PR in the future for changing this behaviour across the integrations.

Reply:
I've made this change now and also exposed the created service in both `useMachine` implementations to match the other integrations.